usdrate/cmd/usdrate-crawler/main.go

```diff
@@ -0,0 +1,249 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/carlescere/scheduler"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/KyberNetwork/tokenrate"
+	"github.com/KyberNetwork/tokenrate/coingecko"
+	"github.com/KyberNetwork/tokenrate/common"
+	"github.com/KyberNetwork/tokenrate/pkg/app"
+	"github.com/KyberNetwork/tokenrate/usdrate/storage"
+)
+
+const (
+	fromTimeFlag       = "from-time"
+	toTimeFlag         = "to-time"
+	jobRunningTimeFlag = "job-running-time"
+	providerFlag       = "provider"
+
+	defaultJobRunningTime = "07:00:00"
+)
+
+func main() {
+	a := app.NewAppWithMode()
+	a.Name = "Token price crawler"
+	a.Usage = "Crawl token price from other exchanges"
+	a.Version = "0.0.1"
+	a.Action = run
+
+	a.Flags = append(a.Flags,
+		cli.StringFlag{
+			Name:   fromTimeFlag,
+			Usage:  "provide from time to crawl token price with format YYYY-MM-DD, e.g: 2019-10-11",
+			EnvVar: "FROM_TIME",
+		},
+		cli.StringFlag{
+			Name:   toTimeFlag,
+			Usage:  "provide to time to crawl token price wiht format YYYY-MM-DD, e.g: 2019-10-12",
+			EnvVar: "TO_TIME",
+		},
+		cli.StringFlag{
+			Name:   jobRunningTimeFlag,
+			Usage:  "crawler will fetch the price daily at this time, e.g: 07:00:00",
+			EnvVar: "JOB_RUNNING_TIME",
+			Value:  defaultJobRunningTime,
+		},
+		cli.StringFlag{
+			Name:   providerFlag,
+			Usage:  "provide provider to get price [coingecko]",
+			EnvVar: "PROVIDER",
+		},
+	)
+	defaultPGDB := "tokenrate"
+	a.Flags = append(a.Flags, coingecko.NewFlags()...)
+	a.Flags = append(a.Flags, app.NewPostgreSQLFlags(defaultPGDB)...)
+	a.Flags = append(a.Flags, app.NewSentryFlags()...)
+	if err := a.Run(os.Args); err != nil {
```

Comment:
sentry flags already added in `NewAppWithMode` so we can remove it here

Reply:
I don't see it in NewAppWithMode?

Reply:
My mistake with `NewApp` function, conversation resolved